SPEC.md

```diff
@@ -98,61 +98,104 @@ A build tool is something which reads one or more input artifacts and writes one
 ## Specifications
 ### Artifact ID
 
-GitBOM uses the git object id (as a blob) for an artifact as its artifact ID.
+A hash function over the []byte of an artifact satisfies the canonical, unique, immutable requirements for an artifact identifier.
+
+### Artifact Identifier Types
+The vast majority of source code artifacts are already indexed by their git object identifiers (gitoids) as git objects of type blob.
```

Comment:
```suggestion
The majority of open source software source code today is already stored in git, and therefore is indexed by a git object identifier ("gitoid") of type "blob".
```

Reply:
I've incorporated the quoting of "gitoid" and "blob".  

Dropped 'vast'.  

Incorporated 'stored in git'